modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -195,7 +195,6 @@ public Swift5ClientCodegen() {
 
         typeMapping = new HashMap<>();
         typeMapping.put("array", "Array");
-        typeMapping.put("List", "Array");
```

Comment:
Why did you delete this?

Reply:
List is not a Swift type. I had a problem while testing [DocuSign specs](https://github.com/docusign/OpenAPI-Specifications) where they define a type `List` and that got mapped to the wrong value `Array` which is a primitive type in swift.

Reply:
Can you elaborate more on this issue please? Perhaps show an example of the issue?

Reply:
I noticed that `List` is not a Swift nor OAS data type. I was working with DS specs and one of their models is named `List`, defined as below.  I was facing a problem where the generator was naming this model `Array` instead of `List`.

```
...
    "list": {
      "type": "object",
      "properties": {
        "anchorAllowWhiteSpaceInCharacters": {
          "description": "",
          "type": "string"
        },
        "anchorAllowWhiteSpaceInCharactersMetadata": {
          "$ref": "#/definitions/propertyMetadata",
          "description": ""
        },
        "anchorCaseSensitive": {
...
```

Reply:
You are right, thanks for this fix and for the extra info 👍